tests/end-to-end/backup-restore-test/backupe2e/backuptest.go

```diff
@@ -3,4 +3,5 @@ package backupe2e
 type BackupTest interface {
 	CreateResources(namespace string)
 	TestResources(namespace string)
+	DeleteResources(namespace string) error
```

Comment:
Unify approach: Do not return error and Use GoConvey, or return errors for every method in `BackupTest` interface